environments/datasets.py

```diff
@@ -0,0 +1,108 @@
+import torch
+from torch.utils.data import Dataset
+from torchvision import transforms
+
+from pendulum import Pendulum
+
+
+class EnvironmentSampler(Dataset):
+    """Dataset for rollout sampling
+    Given an environment and sampling conditions, the dataset samples rollouts as pytorch tensors.
+    """
+
+    def __init__(self, environment, length, number_of_frames, delta_time, number_of_rollouts, img_size, noise_std, radius_bound, world_size, seed, data_mean=.5, data_std=.5):
+        """Constructor method
+
+        Args:
+            environment (Environment): Instance belonging to Environment abstract base class.
+            length (int): Length of the dataset.
+            number_of_frames (int): Total duration of video (in frames).
+            delta_time (float): Frame interval of generated data (in seconds).
+            number_of_rollouts (int): Number of rollouts to generate.
+            img_size (int): Size of the frames (in pixels).
+            color (bool): Whether to have colored or grayscale frames.
+            noise_std (float): Standard deviation of the gaussian noise source, no noise for noise_std=0.
+            radius_bound (float, float): Radius lower and upper bound of the phase state sampling.
+                Init phase states will be sampled from a circle (q, p) of radius
+                r ~ U(radius_bound[0], radius_bound[1]) https://arxiv.org/pdf/1909.13789.pdf (Sec. 4)
+            world_size (float) Spatial extent of the window where the rendering is taking place (in meters).
+            seed (int): Seed for reproducibility.
+            data_mean (float): Data mean for standaritzation. Defaults to 0.5.
```

Comment:
Shouldnt this default to 0 and std dev to 1 to have normalized data?

Reply:
If we have an image that is [0...1] to have standarized data [-1 ... 1] we can use 0.5 as mean and std, that is why i took this default value

Reply:
Oh i see, soory. I thought the images were 0-255